ci/checks/style.sh

```diff
@@ -14,6 +14,11 @@ LANG=C.UTF-8
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
 
+FORMAT_FILE_URL=https://raw.githubusercontent.com/rapidsai/rapids-cmake/main/cmake-format-rapids-cmake.json
+export RAPIDS_CMAKE_FORMAT_FILE=/tmp/rapids_cmake_ci/cmake-formats-rapids-cmake.json
+mkdir -p $(dirname ${RAPIDS_CMAKE_FORMAT_FILE})
+wget -O ${RAPIDS_CMAKE_FORMAT_FILE} ${FORMAT_FILE_URL}
+
```

Comment:
Is there a command somewhere that I'm missing that consumes the `cmake-format-rapids-cmake.json` file? Just trying to understand how all this works.

Reply:
Yes. This PR is building on #9484, which added a script to run cmake-format and cmake-lint along with corresponding pre-commit hooks. One of the important considerations in that PR was that running cmake-format on RAPIDS projects requires the `cmake-format-rapids-cmake.json`, but that file is only downloaded when we build the libraries and it's placed in the build directory, and we did not want pre-commit to fail if a developer did not have the build directory available or if the file was placed at a nonstandard location. So there are pre-commit hooks in place that will run if the file is present, either at a default location or at the user-provided `RAPIDS_CMAKE_FORMAT_FILE` path, but will otherwise be no-ops. Simply downloading the file and setting the `RAPIDS_CMAKE_FORMAT_FILE` environment variable is sufficient in this case to trigger the formatting when pre-commit runs just below this.

Reply:
Great, thanks!